cli/util/convert.go

```diff
@@ -23,6 +31,13 @@ func NewCommands() []cli.Command {
         and converted to other formats. Strings are escaped and output in quotes.`,
 					Action: handleParse,
 				},
+				{
+					Name:      "txdump",
+					Usage:     "Dump transaction stored in file",
+					UsageText: "txdump [-r <endpoint>] --testinvoke <file.in>",
```

Comment:
Why do we need `--testinvoke`? Just dump without `-r` and test-execute with it.

Reply:
Also added a test with `-r` omitted.